contrib/rpm/nexodus.spec.in

```diff
@@ -63,13 +63,13 @@ Requires: nftables
 #%go_generate_buildrequires
 
 %build
-%if 0%{?fedora}
-# has Go 1.20 already
-%else
-  curl -L https://go.dev/dl/go1.20.4.linux-amd64.tar.gz -o go1.20.4.linux-amd64.tar.gz
-  tar -C .. -xzf go1.20.4.linux-amd64.tar.gz
+REQUIRED_GO_VERSION="1.21"
+GO_VERSION=$(go version | awk '{print $3}' | cut -c 3-)
+if [[ ! "$GO_VERSION" == $REQUIRED_GO_VERSION* ]]; then
+  curl -L https://go.dev/dl/go1.21.5.linux-amd64.tar.gz -o go1.21.5.linux-amd64.tar.gz
+  tar -C .. -xzf go1.21.5.linux-amd64.tar.gz
   export PATH=/$(pwd)/../go/bin:$PATH
-%endif
+ fi
```

Comment:
As discussed on slack, I would like to prefer the distro provided go when the version is sufficient. If the package were to ever become more "official," that would be a requirement.

Reply:
I think this is what I was trying to achieve with above change. It's checking if the installed version is 1.21 (which is what we need for this PR), if not than installed the required version. please let me know what am i missing here, I will push the fix.